laura/walkthroughs/week-4-libraries/maps/examples/hello-world/src/main/webapp/index.html

```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
+
 <html>
 <head>
   <title>Hello Google Maps</title>
-  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDlYdo7j26Ndok5V6lRbRfkHQOzAYCL1-0"></script>
```

Comment:
submitting api keys to the source code is not the best idea, ideally they could some from command line arguments, and only applied when using the binary, but fine for now, this is not a production app, etc.